src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exception.CantWriteToFileException;
+import core.basesyntax.service.FileWriterService;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class FileWriterImpl implements FileWriterService {
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String HEADER = "fruit,quantity";

```

Comment:
adding this header is not the responsibility of the file writer, it should just right the data it recieves withouth any modifications to it